src/stories/header/header.scss

```diff
@@ -149,6 +149,12 @@
   ::placeholder {
     color: $c-text-secondary-gray;
   }
+
```

Comment:
I am not sure how this looks "in the wild", but be aware that if it removes the possibility to see if an element is in focus, then it does not comply with WCAG2.1 https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

Reply:
yes @sinejespersen. In the mockups, focus state is shown by changing the background color of the input field to white without it being outlined. So the whole field changes color from:
![image](https://user-images.githubusercontent.com/28546954/169246371-af9fee4d-fea4-4c09-ba41-4bdf40f6e38a.png)
to:
![image](https://user-images.githubusercontent.com/28546954/169246192-822e6b95-4f76-4956-97b5-282669d3690a.png)

Reply:
Sure is, thanks.